src/Api/Doctrine/AgoraExtension.php

```diff
@@ -29,19 +36,47 @@ public function applyToCollection(
         ?Operation $operation = null,
         array $context = [],
     ): void {
-        $this->modifyQuery($queryBuilder, $resourceClass);
+        $this->modifyQuery($queryBuilder, $resourceClass, $context);
     }
 
-    private function modifyQuery(QueryBuilder $queryBuilder, string $resourceClass): void
+    private function modifyQuery(QueryBuilder $queryBuilder, string $resourceClass, array $context = []): void
     {
         if (Agora::class !== $resourceClass) {
             return;
         }
 
         $alias = $queryBuilder->getRootAliases()[0];
-        $queryBuilder
-            ->andWhere("$alias.published = :true")
-            ->setParameter('true', true)
-        ;
+
+        if (PrivatePublicContextBuilder::CONTEXT_PRIVATE !== $context[PrivatePublicContextBuilder::CONTEXT_KEY]) {
+            $queryBuilder
+                ->andWhere("$alias.published = :true")
+                ->setParameter('true', true)
+            ;
```

Comment:
we can stop execution inside this if, no ?  

```suggestion
            ;
            return;
```